src/Java.Interop/Java.Interop/JavaPrimitiveArrays.tt

```diff
@@ -24,7 +24,7 @@ namespace Java.Interop {
 	};
 #>
 	partial class JniRuntime {
-		static readonly KeyValuePair<Type, JniTypeSignature>[] JniBuiltinArrayMappings = new[]{
```

Comment:
Why remove the `new[]` on this and other lines? It's just noise.

Reply:
That just removes unnecessary `new` keyword to make simpler code, similar to using `nameof`. I can remove it.